bindata/network/openshift-sdn/alert-rules.yaml

```diff
@@ -0,0 +1,39 @@
+apiVersion: monitoring.coreos.com/v1
```

Comment:
This needs the magic annotation that makes the network operator not care if creation fails.